core/controllers/feedback_test.py

```diff
@@ -118,8 +120,12 @@ class FeedbackThreadIntegrationTests(test_utils.GenericTestBase):
     def setUp(self):
         super(FeedbackThreadIntegrationTests, self).setUp()
         self.signup(self.EDITOR_EMAIL, self.EDITOR_USERNAME)
-        self.EDITOR_ID = self.get_user_id_from_email(self.EDITOR_EMAIL)
-
+        self.signup(self.VIEWER_EMAIL, self.VIEWER_USERNAME)
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
```

Comment:
nit: can you follow the order "owner, editor, viewer", as in the three lines below?